backend/database/versions/33_95ac900d53ef_redesign.py

```diff
@@ -0,0 +1,99 @@
+"""
+
+Revision ID: 33_95ac900d53ef
+Revises: 32_c27083d1a76d_schema
+Create Date: 2023-01-03 15:11:24.868061
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "95ac900d53ef"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "Collection",
+        sa.Column("id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column("version_id", postgresql.UUID(as_uuid=True), nullable=True),
+        sa.Column("originally_published_at", sa.DateTime(), nullable=True),
+        sa.Column("tombstoned", sa.BOOLEAN(), nullable=True),
+        sa.PrimaryKeyConstraint("id"),
+        schema="persistence_schema",
```

Comment:
I believe the schema`persistence_schema` also needs to be created, although I am not sure.

Reply:
Added it